.github/workflows/test.yml

```diff
@@ -15,6 +15,16 @@ jobs:
           github_token: ${{ secrets.github_token }}
           reporter: github-check
           level: info
+          workdir: ./testdata/
+      - name: Check if code is left untouched (not formatted)
```

Comment:
I think we can remove this test if the action doesn't intend to change the code in the first place.
I don't think we need to copy&past this test for each reporter at least.

Reply:
@haya14busa Your right the remark linter doesn't change the files unless the user explicitly adds the `--output` flag. This is therfore default behavoir which we don't need to check.